### ISSUE TITLE
Fix Sidekiq pipelining deprecation warnings

### ISSUE THREAD

**Issue by apertome** [avalonmediasystem/avalon#4612]
Fix Sidekiq pipelining deprecation warnings

Upgrading to sidekiq 6.4.1 should fix this (we're currently on 6.4.0).

The warnings look like this:

```
worker_1         | Pipelining commands on a Redis instance is deprecated and will be removed in Redis 5.0.0.
worker_1         |                   
worker_1         | redis.multi do    
worker_1         |   redis.get("key")
worker_1         | end
worker_1         |                      
worker_1         | should be replaced by
worker_1         |                              
worker_1         | redis.multi do |pipeline|
worker_1         |   pipeline.get("key")
worker_1         | end
worker_1         |                                                                                                
worker_1         | (called from /usr/local/bundle/gems/sidekiq-6.4.0/lib/sidekiq/launcher.rb:141:in `block in ❤'}
worker_1         | Pipelining commands on a Redis instance is deprecated and will be removed in Redis 5.0.0.
worker_1         |                   
worker_1         | redis.multi do    
worker_1         |   redis.get("key")
worker_1         | end                                                                                         
worker_1         |                      
worker_1         | should be replaced by
worker_1         |                              
worker_1         | redis.multi do |pipeline|
worker_1         |   pipeline.get("key")
worker_1         | end
worker_1         |                                                                                                    
worker_1         | (called from /usr/local/bundle/gems/sidekiq-6.4.0/lib/sidekiq/launcher.rb:164:in `block in ❤'}                           
worker_1         | Pipelining commands on a Redis instance is deprecated and will be removed in Redis 5.0.0.
worker_1         |                                                                                                    
worker_1         | redis.pipelined do                                                                       
worker_1         |   redis.get("key")
worker_1         | end                                                                                         
worker_1         |                   
worker_1         | should be replaced by
worker_1         |
worker_1         | redis.pipelined do |pipeline|
worker_1         |   pipeline.get("key")
worker_1         | end
worker_1         |
worker_1         | (called from /usr/local/bundle/gems/sidekiq-cron-1.2.0/lib/sidekiq/cron/job.rb:206:in `block in all'}
worker_1         | Pipelining commands on a Redis instance is deprecated and will be removed in Redis 5.0.0.
```

@cjcolvar noted that there is a sidekiq issue about this:
https://github.com/mperham/sidekiq/issues/5202

**Comment by apertome** [comment]
The deprecation warnings coming from `sidekiq` have been addressed by upgrading to sidekiq version `6.4.1`

However, there were other warnings coming from `sidekiq-cron`, which appears to be unmaintained:
https://github.com/ondrejbartas/sidekiq-cron/issues/310

There was a contributed fix in the master branch of `sidekiq-cron` repo, but it hasn't been released and may never be. For now, I created our own fork of `sidekiq-cron` and pulled that fix into a branch based on the `1.2.0` tag, which was the last released `sidekiq-cron` version.

**Comment by joncameron** [comment]
This is also now resolved.
